recipe/meta.yaml

```diff
@@ -15,15 +15,15 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.7
     - pip
   run:
-    - python >=3.6
-    - geopandas-base >=0.9
-    - dask >=2.2.0,!=2021.05.1
-    - distributed >=2.2.0,!=2021.05.1
-    - numba
+    - python >=3.7
+    - geopandas-base >=0.10
+    - dask >=2021.06.0
+    - distributed >=2021.06.0
```

Comment:
Similarly to geopandas-base, we could also change `dask` to `dask-core` (and remove `distributed`) to make a lighter install possible (https://github.com/conda-forge/dask-feedstock/blob/master/recipe/meta.yaml)

Reply:
That sounds good. I actually think that we should not list `distributed` among dependencies at all, we do not import it directly anywhere.